pkgs/development/python-modules/digi-xbee/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchPypi, pyserial, srp, lib }:
+
+buildPythonPackage rec {
+  pname = "digi-xbee";
+  version = "1.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2ed798faee0853bf7ae9ca5aa4bdcbab496e3c2d56c9f0719a8e3e0d13270891";
+  };
+
+  propagatedBuildInputs = [ pyserial srp ];
+
+  # Package doesn't distribute tests.
+  doCheck = false;
```

Comment:
Looks like they do releases on github, would prefer to have unit tests from github over not having them at all.

Reply:
I will make sure that the tests aren't bound to any hardware before enabling! But yes indeed, good point! My first package ^_^!

Reply:
The tests are functionnal tests that requires to have an xbee radio connected to the PC which might not be wanted if you only want to have to access to the library for a test or two.

Reply:
If they don't have any unit tests, then just update the comment saying, "# Upstream doesn't contain unit tests, only functional tests which require specific hardware"

Reply:
ok good! Will do! Thank you!